packages/host/src/DetailPanel.tsx

```diff
@@ -72,16 +70,16 @@ const StyledAccordian = styled(Accordion)(({ theme }) => ({
 }));
 
 const DetailPanel: React.FC = () => {
-  const { close, isOpen, open, sections } = useDetailPanelStore();
+  const { actions, close, isOpen, open, sections } = useDetailPanelStore();
   const t = useTranslation();
   const theme = useTheme();
   const isSmallScreen = useMediaQuery(theme.breakpoints.down('lg'));
 
   const Sections = useCallback(
     () => (
       <>
-        {sections.map(section => (
-          <Box key={section.titleKey}>
+        {sections.map((section, index) => (
+          <Box key={`action.titleKey_${index}`}>
```

Comment:
a little dodgy - entirely so that we can have duplicate section titles without having to manually specify a key

Reply:
sometimes you gotta use an index 🤷